packages/core-mobile/app/store/rpc/handlers/avalanche_bridgeAsset/avalanche_bridgeAsset.ts

```diff
@@ -131,7 +132,8 @@ class AvalancheBridgeAssetHandler
           isTestnet: isDeveloperMode,
           onStatusChange: noop,
           onTxHashChange: noop,
-          request
+          request,
+          activeNetwork
```

Comment:
you probably should not pass active network here in case the request is for another network. inside  BridgeService.transferEVM() there is already a network object that I think you can use to get the provider. @onghwan do you agree?

Reply:
you are right.